homeassistant/components/philips_air_purifier/__init__.py

```diff
@@ -0,0 +1,37 @@
+"""The Philips Air Purifier integration."""
+from __future__ import annotations
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .client import ReliableClient
+from .const import COAP_PORT, DOMAIN
+
+PLATFORMS: list[str] = ["fan"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Philips Air Purifier from a config entry."""
+
+    client = ReliableClient(hass.loop, entry.data["host"], port=COAP_PORT)
+    client.start()
+    if DOMAIN not in hass.data:
+        hass.data[DOMAIN] = {}
+    hass.data[DOMAIN][entry.entry_id] = client
+
+    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
+
+    return True
+
+
+async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Unload a config entry."""
+    unload_ok = await hass.config_entries.async_unload_platforms(entry, PLATFORMS)
+
+    client = hass.data[DOMAIN][entry.entry_id]
+    client.stop()
```

Comment:
```suggestion
```

Reply:
I've moved the device_id and device_info to the constructor.

For the name - I think it can change over time, so I'd prefer to use the name received from the device when it's available.